MatrixKit/Categories/NSBundle+MatrixKit.m

```diff
@@ -15,16 +15,17 @@
  */
 
 #import "NSBundle+MatrixKit.h"
+#import "MXKViewController.h"
 
 @implementation NSBundle (MatrixKit)
 
 static NSString *customLocalizedStringTableName = nil;
 
 + (NSBundle*)mxk_assetsBundle
 {
-    NSString *bundleResourcePath = [NSBundle mainBundle].resourcePath;
-    NSString *assetPath = [bundleResourcePath stringByAppendingPathComponent:@"MatrixKitAssets.bundle"];
-    return [NSBundle bundleWithPath:assetPath];
+    NSBundle * bundle = [NSBundle bundleForClass:[MXKViewController class]];
```

Comment:
With cocoa pods 1.x.x, the pod ressources are not copied in the main bundle, they are in the pod bundle. 
So if you are using cocoapods 1.x.x, you cannot load MatrixKit assets by using mainBundle